### ISSUE TITLE
vega-lite 5

### ISSUE THREAD

**Issue by davidanthoff** [nteract/nteract#5474]
Vega-lite 5 is out, and it would be great if we could add support for it (and the new MIME type for it) to nteract :) I think nteract had this awesome system, where it essentially can support all vega and vega-lite versions at the same time, right? So fingers crossed this might be a relatively simple change?

CC @domoritz

**Comment by domoritz** [comment]
We made sure most of Vega-Lite 5 is backwards compatible (even if the schema is not backwards compatible for example for selections).

**Comment by davidanthoff** [comment]
I think here in nteract there is all the infrastructure there already so that we can just add support for a new vega-lite-5 MIME type, and then it ships the latest vega-lite 4 library for v4 MIME types, and vega-lite 5 library for v5 MIME types. So at least on the nteract side of things I believe backwards compat is actually not an issue :)

**Comment by domoritz** [comment]
Even better!

**Comment by captainsafia** [comment]
Yep -- the infrastructure for vega integration is over at the https://github.com/nteract/any-vega repo. 

@CrystallineCat authored the package but I believe what's required is adding the transpiled JS sources for the new version and updating the https://github.com/nteract/any-vega/blob/472f50dd64c3b72f4a4f10db4c6ac4e963baeac8/packages/any-vega/index.ts file with the new media type and export.

**Comment by davidanthoff** [comment]
Bump :) We would like to ship vega-lite 5 in https://github.com/queryverse/VegaLite.jl but we can only do that once the various clients start to support the vega-lite 5 MIME type and also ship vega-lite 5. I don't understand the code here enough to open a PR, but would be fantastic if we could try to ship vega-lite 5 with nteract!

**Comment by davidanthoff** [comment]
@CrystallineCat is there a chance you could help us with this? I've stared at it for a while now, but I just don't understand what I would have to do to get this to work...

I think ideally we would a) add support for vega-lite 5, b) the new MIME type that corresponds to vega-lite 5, and c) I think generally update the vega and vega-lite versions that ship. On c), for example there are newer versions of vega 5 and vega-lite 4 than the ones referenced in `any-vega`.

And finally, it would be awesome if maybe @CrystallineCat could update the README of `any-vega` a little bit with instructions on how to keep things up-to-date. I'd be more than happy to occasionally bump some version numbers here and there, but at the moment I just have no clue where and how :) Just some short instructions a la "To update, make a change in file X, run tool Y and commit" or something like that would go a long way.

Thanks :)

**Comment by willingc** [comment]
Added an issue at https://github.com/nteract/any-vega/issues/5

**Comment by captainsafia** [comment]
CrystallineCat isn't active on the project at the moment so we'll have to coordinate next steps for the moment.

> Added an issue at nteract/any-vega#5

Let's go ahead and transfer the issue so we can retain the context.
